HOWTO-make-a-release.md

```diff
@@ -137,19 +143,27 @@ You will need to checkout at least three working trees:
 
 ## Freeze the source repository
 
-Three business day before the release, freeze the main repository.  This
-locks out everyone but the named user, who is doing the release, from doing
-any pushes.  Someone other than the person doing the release should run the
-command.
+Three business day before the release, freeze the appropriate source
+repository.
```

Comment:
At the moment we only freeze the main repo. Is it possible to freeze the premium repo?

Reply:
Yes.  I checked, the frozen hook is enabled

Reply:
Not that we really need to freeze it given the frequency of pushes there and that we push there only on particular circumstances.

Reply:
So.....  should I modify this, or leave it as is?

Reply:
We've never previously frozen it, so I think lets not bother doing this for premium. It seems over the top.